src/pybind/mgr/rook/module.py

```diff
@@ -305,7 +305,7 @@ def describe_service(self,
             for zone in all_zones.get('items', []):
                 rgw_realm = zone['metadata']['name']
                 rgw_zone = rgw_realm
-                svc = 'rgw.' + rgw_realm + '.' + rgw_zone
+                svc = 'rgw.' + rgw_realm
```

Comment:
but i think something is awry with the rgw service naming.  the orch interface now lets you name the service whatever you like--it's not directly tied to the realm or zone.